src/Symfony/Bridge/Doctrine/DataCollector/DoctrineDataCollector.php

```diff
@@ -138,6 +138,9 @@ private function sanitizeQuery($connectionName, $query)
         if (!\is_array($query['params'])) {
             $query['params'] = [$query['params']];
         }
+        if (!\is_array($query['types'])) {
```

Comment:
I don't how is this method called. Could be `ArrayAccess` a valid type for `$query`?

Reply:
I don't think so. Queries are taken from dbal DebugStack logger which has this logic:
```
public function startQuery($sql, ?array $params = null, ?array $types = null)
{
    $this->queries[++$this->currentQuery] = ['sql' => $sql, 'params' => $params, 'types' => $types, 'executionMS' => 0];
}
```